tests/app/dao/test_service_sms_sender_dao.py

```diff
@@ -164,11 +164,10 @@ def test_dao_update_service_sms_sender_switches_default(notify_db_session):
         .order_by(ServiceSmsSender.created_at)
```

Comment:
```suggestion
        .order_by(ServiceSmsSender.created_at)
```
^ remove this I guess